NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -87,10 +85,10 @@ - (void) run {
     NSBundle *bundle = [NSBundle mainBundle];
     NSString *path = [bundle pathForResource:@"hosts" ofType:@"txt"];
     mk::ooni::DnsInjectionTest()
-        .set_backend("8.8.8.8:53")
+        .set_options("backend", "8.8.8.1:53")
         .set_input_file_path([path UTF8String])
-        .set_verbose()
-        .on_log([self](const char *s) {
+        .set_verbosity(MK_LOG_DEBUG2)
+        .on_log([self](uint32_t, const char *s) {
```

Comment:
What you see above is what changed in the API. Below there would be more noise because I also reindented.